DATA/production/qc-sync/mft.json

```diff
@@ -82,26 +82,52 @@
         "remoteMachine": "alio2-cr1-qme05.cern.ch",
         "remotePort": "47797",
         "localControl": "odc"
+      },
+      "MFTAsyncTask": {
```

Comment:
This task is called "Async", are you sure you want it in the "sync" json?

Reply:
Hi @chiarazampolli, the naming here is for historical reasons. Originally there were no tracks reconstructed online for muons (MFT, MCH, MID) but with the latest developments there is part of the MUON tracks reconstructed for ED and QC. For this reason we now include the task in sync QC but it is still called async task. We will come up with a new naming scheme but for now we kept the old name. 

Also I have been told during past discussions that the json file in the sync folder which I am modifying in this PR is used only for local tests and nothing else. Please let me know if this is not true. 

The file which is being actually used at P2 is located in Consul: https://ali-consul-ui.cern.ch/ui/alice-o2-cluster/kv/o2/components/qc/ANY/any/mft-track-full-qcmn/edit and we have been using that for many weeks now. 
This PR is just copy paste of the file from consul.

Reply:
correct, the file here is only for the full system test in the CI and for local tests.
Operation at P2 takes the file from consul as defined here: https://github.com/AliceO2Group/O2DPG/blob/master/DATA/production/qc-workflow.sh#L14 (compare `EPNSYNCMODE=1` (P2) v.s. `SYNCMODE=1` (Sync FST))

Reply:
Great, thank you for confirming.

Reply:
Sure, my point was more that if this is the copy of the one used in sync, and we had there something for async, it would have been wrong. But after the explanation by @tomas-herman , all is clear.